ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.less

```diff
@@ -20,14 +21,20 @@
     }
 
     .search-form-label {
-        .is-bold();
+        font-weight: bold;
```

Comment:
why this change?  `.is-bold()` was recommended over font-weight

Reply:
put back to .is-bold()